code/ietf-sid-file.yangtree

```diff
@@ -12,6 +12,7 @@ module: ietf-sid-file
     |  +-- entry-point    sid
     |  +-- size           uint64
     +-- item* [namespace identifier]
+       +-- status?       string
```

Comment:
tree diagram not refreshed with sid-file-status

Reply:
Done in 82fbed5